NIPTool/API/external/api/api_v1/endpoints/statistics.py

```diff
@@ -25,24 +26,21 @@ def statistics(request: Request, adapter: NiptAdapter = Depends(get_nipt_adapter
     ]
 
     batches = get_last_batches(adapter=adapter, nr=nr_batches)
```

Comment:
This should be `find.get_last_batches()` and return `List[Batches]`

Reply:
this is imported from utils. Lets make utils better in a separate PR